tools/pvacseq/generate_protein_fasta.py

```diff
@@ -64,11 +68,19 @@ def convert_vcf(input_vcf, temp_dir, sample_name):
     }
     if sample_name is not None:
         convert_params['sample_name'] = sample_name
+    if phased_proximal_variants_vcf is not None:
+        convert_params['proximal_variants_vcf'] = phased_proximal_variants_vcf
+        proximal_variants_tsv = os.path.join(temp_dir, 'proximal_variants.tsv')
+        convert_params['proximal_variants_tsv'] = proximal_variants_tsv
+        convert_params['flanking_bases'] = flanking_sequence_length * 4
```

Comment:
convert_params['flanking_bases'] = flanking_sequence_length * 4
I do ask myself if this is correct? Shouldn't it be  "flanking_sequence_length * 3" ?
An AA is encoded by 3 bases, right?

Reply:
Yes, we are just giving ourselves a bit of buffer here.

Reply:
OK.